example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 31
```

Comment:
Will this cause any backwards compatibility issues?

Reply:
No this is just the target version. It shouldn't block any users on old Android versions